### PR TITLE
Pedestrian option adjustments 

### PR DESCRIPTION
**Pull request by kdiluca**
# Issue

What issue is this PR targeting? If there is no issue that addresses the problem, please open a corresponding issue and link it here.

## Tasklist

 - [ ] Add tests
 - [ ] Review - you must request approval to merge any PR to master
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Generally use squash merge to rebase and clean comments before merging
 - [ ] Update the [changelog](CHANGELOG.md)
 - [ ] Update relevant [documentation](docs/README.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: src/sif/pedestriancost.cc [hunk 1]
```diff
@@ -50,7 +50,7 @@ constexpr uint32_t kDefaultMaxGradeWheelchair = 12; // Conservative for now...
 constexpr uint8_t kDefaultMaxHikingDifficulty = 1; // T1 (kHiking)
 constexpr float kModeFactor = 1.5f;                // Favor this mode?
 constexpr float kDefaultWalkwayFactor = 0.9f;      // Slightly favor walkways
-constexpr float kDefaultSideWalkFactor = 0.95f;    // Slightly favor sidewalks
+constexpr float kDefaultSideWalkFactor = 0.25f;    // Slightly favor sidewalks
```
Comment (dnesbitt61) [inline src/sif/pedestriancost.cc]: my experience is that factors this low can cause issues. Does this not work at a value of 0.9 or 0.8?

File: src/sif/pedestriancost.cc [hunk 2]
```diff
@@ -650,13 +650,13 @@ Cost PedestrianCost::EdgeCost(const baldr::DirectedEdge* edge, const uint32_t sp
 
   // TODO - consider using an array of "use factors" to avoid this conditional
   float factor = 1.0f + kSacScaleCostFactor[static_cast<uint8_t>(edge->sac_scale())];
-  if (edge->use() == Use::kFootway) {
+  if (edge->use() == Use::kFootway or edge->use() == Use::kSidewalk) {
```
Comment (kevinkreiser) [inline src/sif/pedestriancost.cc]: wrong language here, you need `||`

Comment (kdiluca) [reply src/sif/pedestriancost.cc]: :man_facepalming:

**Comment by kdiluca** [comment]
I'd like to be able to merge this to master and push to valhalla staging tomorrow AM so that @dgearhart can do some side-by-side compares with the shared guidance tools.  Also, it's been awhile since we're tagged a release so we'll need to update the change log for this as well.

**Comment by kdiluca** [comment]
I'd like to be able to merge this to master so that @dgearhart can do some RAD testing. Also, it's been awhile since we've tagged a release so we'll need to update the change log for this as well.

**Comment by kdiluca** [comment]
Pedestrian Diffs:  (pedestrian_routes.txt)
Valhalla With sidewalk adjustment, no walkway_factor:
#87 
image
image
vs.
Valhalla With sidewalk adjustment, walkway_factor=1.5:
#88 
image
image
image


Valhalla With sidewalk adjustment, no walkway_factor:
#90  
image
image
vs.
Valhalla With sidewalk adjustment, walkway_factor=1.5:
#91
image
image

**Pull request merged**